projectgenerator/libqgsprojectgen/generator/postgres/postgrescreator.py

```diff
@@ -187,6 +188,17 @@ def layers(self):
                     field.widget = 'TextEdit'
                     field.widget_config['IsMultiline'] = True
 
+                if 'time' in fielddef['data_type'] or 'date' in fielddef['data_type']:
+                    field.widget = 'DateTime'
+                    field.widget_config['calendar_popup'] = True
+                    # timestamps don't need further configuration
+                    if 'time' in fielddef['data_type'] and not 'timestamp' in fielddef['data_type']:
+                        field.widget_config['field_format'] = 'HH:mm:ss'
+                        field.widget_config['display_format'] = 'HH:mm:ss'
+                    elif 'date' in fielddef['data_type']:
+                        field.widget_config['field_format'] = 'yyyy-MM-dd'
+                        field.widget_config['display_format'] = 'yyyy-MM-dd'
```

Comment:
We can use 2017-08-24, but it's more common 24-08-2017. We never use 2017.08.24 nor 24.08.2017, although I like them :).